onnxruntime/python/tools/transformers/models/whisper/benchmark.py

```diff
@@ -483,20 +506,29 @@ def parse_args():
     parser.add_argument("--pt-num-rows", type=int, default=1000, help="Number of rows for PyTorch profiler to display")
     parser.add_argument("--verbose", default=False, action="store_true")
     parser.add_argument("--log-folder", type=str, default=os.path.join("."), help="Folder to cache log files")
+    parser.add_argument("--tune", default=False, action="store_true")
```

Comment:
Can you add a `help` argument that mentions that `--tune` is for ROCm?

Reply:
Done.